conversions/bytes_to_int.py

```diff
@@ -0,0 +1,64 @@
+def bytes_to_int(
+    bytes_var: bytearray,
+    byteorder: str,
+    signed: bool,
+) -> (int):
+
+    r"""
+    This script first converts a bytearray,
+    to its corresponding signed/unsigned binary number,
+    and then converts the binary number to an integer.
+
+    >>> tests = ((b"\x00\x10", "big", False), (b'\xfc\x00', "big", True),
+    ...          (b'\x00\x01', "big", False), (b'\x00\x10', 'little', False))
+    >>> all(bytes_to_int(a,b,c) == int.from_bytes(a,b,signed=c)
+    ...     for a,b,c in tests)
+    True
+    >>> bytes_to_int('abc',False,'big')
+    AttributeError - 'bool' object has no attribute 'lower'
+    0
+    >>> bytes_to_int(7.1, 'little', signed=True)
+    TypeError - 'float' object is not subscriptable
+    0
+
+    """
+
+    try:
```

Comment:
`try` blocks should be small to make code easier to debug.

Allow your code to raise the exceptions.  If raising exceptions on bad input data is good enough for the Python Standard Library, then it is good enough for `bytes_to_int()`.